Marlin/src/gcode/calibrate/G34_M422.cpp

```diff
@@ -45,56 +45,54 @@
 #define DEBUG_OUT ENABLED(DEBUG_LEVELING_FEATURE)
 #include "../../core/debug_out.h"
 
-//
-// Sanity check G34 / M422 settings
-//
-constexpr xy_pos_t test_z_stepper_align_xy[] = Z_STEPPER_ALIGN_XY;
 
-#if ENABLED(Z_STEPPER_ALIGN_KNOWN_STEPPER_POSITIONS)
+#if defined(Z_STEPPER_ALIGN_XY)
```

Comment:
There are some issues related to Z_STEPPER_ALIGN_KNOWN_STEPPER_POSITIONS. I'll send you a PR to fix compilation for that use case, although I've not actually tested them yet.